codespell_lib/data/dictionary.txt

```diff
@@ -4016,10 +4017,16 @@ certifactes->certificates
 certifcate->certificate
 certifcates->certificates
 certifcation->certification
+certifciate->certificate
+certifciates->certificates
 certificat->certificate
+certificats->certificates
 certifice->certificate
+certifices->certificates
+certifificate->certificate
 certifificates->certificates
 certiticate->certificate
+certiticates->certificates
```

Comment:
Can you add the -ed and -ion variants of these ones too please?

Reply:
6b5626ca558be44bc06f48e61e85c44dff74bd06